### ISSUE TITLE
WCAG 2.1: Reflow - Accessing 'how to test' content in 'tab stops' requires two dimensional scrolling at 200% zoom

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#771]
**To Reproduce**
Adjust the following display settings in Windows :
- Display resolution to ```1280 * 1024```
- Scaling size to ```200%```

1.  Launch AI-Windows
2. Navigate to Tests->Tabstops
3. Observe two dimensional scrolling is required to access all the "How to test" content

**Expected behavior**
Text content should be made available without requiring to scroll in two dimensions

**Actual behavior**
Text content requires tow dimensional scrolling

**Screenshots or .GIF**
![Screenshot (309)](https://user-images.githubusercontent.com/22068434/80002818-ff266300-8474-11ea-974e-ffa57c6614f1.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise, Version1903
 - Accessibility Insights for Windows Version: Production Release: v1.1.1198.01

**Comment by adarshrema** [comment]
From internal testing for accessibility. To be combined with results from planned HC testing.

**Comment by juchampa** [comment]
@iamrafan This is no longer in scope

**Comment by juchampa** [comment]
@iamrafan this item is out of scope
